mantle/Makefile

```diff
@@ -13,7 +13,8 @@ schema-update:
 .PHONY: install
 install: bin/ore bin/kola bin/plume bin/kolet
```

Comment:
The dependencies are stale, and should probably just be `build` now.

Reply:
updated it to just `install: build` which is what I think you wanted